src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -280,6 +281,32 @@ public static SimpleInterval trimInterval(final Locatable interval,
         }
     }
 
+    /** Read only the sample names from a segment reader.
+     *
+     * @param segmentReader Never {@code null}
+     * @param segmentName Never {@code null}
+     * @return a list of the unique sample names in the seg file.
+     */
+    public static List<String> readSampleNamesFromSegmentReader(final Reader segmentReader, final String segmentName) {
```

Comment:
Done. Also, updated the javadoc. `segmentName` renamed to `segmentSourceName`. It is a string identifier for `Reader` (it is only used in error messages).